### ISSUE TITLE
Masthead :- Accessibility :- Search edit combo list box

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#3158]
Issue Description :- No label for this control. 

Recommended Fix:- Give label as "search".

Happening on :- Voiceover iPhone JAWS FF

URL : 
http://ibmdotcom-web-components-canary.mybluemix.net/?
